future_challenges_for_RSEng.tex

```diff
@@ -67,10 +70,83 @@ \section{Aspects of this future}
 \subsection{What do we want to have achieved by 2035}
 
 \subsection{Definition and Evolution of RSEs}
-How will the Definition of an RSE evolve in ten years?
+\begin{framed}
+How will the definition of an RSE evolve in ten years?
+
 What will not change in the essence of an RSE ten years from now?
-Here we consider how the topics of RSEng will change in the face of the digitalization of society.
-Also here: What will our topics be after version control doesn’t need to be taught anymore? Digitalisation will also feature more heavily in the domain curricula.
+\end{framed}
+% Here we consider how the topics of RSEng will change in the face of the digitalization of society.
+% Also here: What will our topics be after version control doesn't need to be taught anymore?
+% Digitalisation will also feature more heavily in the domain curricula.
+
+The digital skills of domain scientists will increase,
+and software will become easier to interact with,
+reducing reliance on RSEs to train newcomers on the most basic tasks
+(JN: for example, versioning and archiving?).
+User training sessions might focus on more advanced topics
+that cater to a smaller audience, or involve mentoring more frequently.
+
+Regarding the definition of RSEs, we will still have a spectrum of competencies,
+from full professional RSEs to domain scientists who develop research software.
+While the RSE core competencies\cite{Goth2024} might remain unchanged,
+there might be a shift in the responsibilities of RSEs,
+in the form of a partial reallocation of the training budget
+to the other core competencies.
+
+What will \emph{not} change over that timeframe is the RSE's ability to remain
+flexible and embrace change. Keep an eye open and pick up tools and skills
+as needed as we go along.
+Communication and collaboration skills will remain essential to engage
+with domain scientists and help them find technical solutions.
```

Comment:
I'm not sure that we currently teach RSEs these communication and collaboration skills (at least not in a structured manner)? If that's true then I guess we just rely on the enthusiasm of people who undertake RSE work to want to engage with and understand the techincal elements of the work being undertaken by the people they work with, and to find novel, high-quality solutions to technical challenges? We should perhaps think about whether we can support continued growth of the RSE community by actively helping to provide training in these skills?